schema/2.0.0/corpora_schema.md

```diff
@@ -39,6 +39,12 @@ This document is organized by:
 
 * **AnnData** - The canonical data format for the cellxgene Data Portal is HDF5-backed [AnnData](https://anndata.readthedocs.io/en/latest) as written by version 0.7 of the anndata library.  Part of the rationale for selecting this format is to allow cellxgene to access both the data and metadata within a single file. The schema requirements and definitions for the AnnData `X`, `uns`, `obs`, and `obsm` attributes are described below.
 
+* **Organisms**. Datasets MUST contain either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal. 
+
+* **Name Collisions in Metadata**. If a metadata key exists in the original dataset with the exact name required by the schema but with values that cannot be accurately mapped to the schema, `"key"` MUST be renamed to `"key_original"`. For example, if there is a `"cell_type"` in the original dataset with values that cannot be mapped, then it MUST be renamed to `"cell_type_original"`. 
+
+* **Redundant Metadata**. If a metadata key exists in the original dataset with a different name than required by the schema and with values that can be accurately mapped to the schema, it is STRONGLY RECOMMENDED that Curators request the permission of the author(s) to rename that key and remodel its values as needed to match the key and values in the schema to eliminate redundant metadata. Common cases are `"Sex"` instead of `"sex"` and `"Age"` instead of `"development_stage"`.
```

Comment:
Age & development stage are not typically considered synonymous, and usually have different values.

I'm surprised to see this document make recommendations for interactions between curators & researchers. I would have expected submission-specific guidance, something closer to "It is STRONGLY RECOMMENDED to avoid multiple metadata fields containing identical or similar information."
But perhaps I'm reading it from the wrong lens?

Reply:
| Age & development stage are not typically considered synonymous, and usually have different values.

I've observed multiple cases in the data corpus where these were duplicates or close. Perhaps curators gravitated to Age as the source for development_stage.

> It is STRONGLY RECOMMENDED to avoid multiple metadata fields containing identical or similar information.

Nice. This is definitely more succinct but it still may be useful to offer some examples.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/ff6c651fd9709f8a074d7d2d7faec4562b1778d1